docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -125,7 +125,9 @@ join output rows can be skipped.
 
 Use connector provided table node partitioning when reading tables.
 For example, table node partitioning corresponds to Hive table buckets.
-When set to ``true`` each table partition is read by a separate worker.
+When set to ``true`` and minimal bucket to task ratio, set
+in ``preferred-write-partitioning-min-number-of-partitions`` is matched or exceeded,
```

Comment:
the comma should be after property name, before "is"

Reply:
this is based on `docs/src/main/sphinx/admin/properties-writer-scaling.rst:37`

Reply:
Use something like this to make it simpler to understand:

```
When set to ``true`` and minimal partition to task ratio is matched or exceeded, each table partition is read by a separate worker. The minimal ratio is defined in ``preferred-write-partitioning-min-number-of-partitions``.
```